src/test/resources/features/Device.feature

```diff
@@ -0,0 +1,75 @@
+Feature: Testing Device API
+
+  Scenario Outline: Create Device
+    Given I want to create a device
+    When I create a device
+    Then I should get a response with status <code>
+    And I should get a response with body
+
+    Examples:
+      | code |
+      | 200  |
+
+  Scenario Outline: Create Same Device
```

Comment:
From this test I understand that the database isn't cleared after each test, is that accurate? In my mind it should be to avoid leaky tests, what do you think?

Reply:
The database is only created once. 
If you create a database after each test:
1. It will be very heavy and it will take long
2. we want to try to simulate a realistic environment where a db is only created once.